apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -793,10 +804,21 @@ describe('Meter', () => {
                 unitName,
             })
 
-            await updateTestProperty(adminClient, property.id, { deletedAt: 'true' })
+            await PropertyAPI.softDelete(adminClient, property.id)
+
+            // NOTE: give worker some time
+            await sleep(1000)
```

Comment:
it looks like another test case! please add another test with sleep

Reply:
done